.github/workflows/build.yml

```diff
@@ -2,8 +2,12 @@ name: 🔨 Build scikit-decide
 
 on:
   push:
+    tags-ignore:
+      - 'nightly'
```

Comment:
With the current filter in master `branches: - "**"` this is not needed as it will never be triggered on tags (which is the expected behaviour, only build for push on branches, not on any tags)